### DISCUSSION TITLE
control page when using switchToProfile event

### DISCUSSION THREAD

**Discussion by mekhaelt** [elgatosf/elgato#59] [Help]
As the name suggests I am trying to control the page the profile is on when using switchToProfile. I have noticed that this is a feature with the switching profile button and was wondering if there was a way to do it with the switchToProfile event

**Comment by GeekyEggo** [comment]
Hey @mekhaelt, this is a really nice suggestion, thank you. I have forwarded it to our Stream Deck team for their consideration. 🙌
